pages/profile/[username].tsx

```diff
@@ -14,50 +15,79 @@ import Head from "next/head"
 
 import Lottie from "react-lottie"
 import animationData from "../../lottie/sad.json"
-const defaultOptions = {
+import {ProfileSerializer} from "types/backend"
+const defaultOptions: {
+    loop: boolean
+    autoplay: boolean
+    animationData: any | null
+} = {
     loop: true,
     autoplay: true,
     animationData: animationData,
     // rendererSettings: {
     // 	preserveAspectRatio: 'xMidYMid slice'
     // }
 }
+export interface Auth {
+    token?: string | null
+    profile?: any | null
+    loading?: boolean
+}
```

Comment:
No need after #102 is merged. Till then, keep this.

Reply:
Since the PR is merged now, you can now take a pull and update this file accordingly.

Reply:
Done